diagnostic.md

```diff
@@ -100,11 +126,21 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH http://www.example.com/donuts/$ID
+--header "Content-Type: application/json" \
+--data '{
+  "name": "Krüller"
+  }'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function(id, data) {
+  return $.ajax({
+    url: config.apiOrigin + '/books/' + id,
+    method: 'PATCH',
+    data: data,
+  });
+};
```

Comment:
Keep in mind, you would ned to make sure data is passing the correct data from your click handler. If not, hard coding the data like this would definitely work. 

```js
let changeDonut = function (donutID) {
  return $.ajax({
    url: 'http://example.com/donuts/' + donutID,
    method: 'PATCH',
    data: {
      "donut" {
        "name": "Krüller",
      }
    }
  });
};
```